qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -61,6 +64,9 @@ def __init__(
                 in http://arxiv.org/abs/2009.10095.
             initial_point: An optional initial point (i.e. initial parameter values)
                 for the optimizer. If ``None`` a random vector is used.
+            alpha: The fraction of top measurement samples to be used for the expectation value
```

Comment:
Is there any constraints of `alpha` (lower or upper bound)? If so, it's good to mention them here and check it in the code as well as you check `reps < 1`.

Reply:
Yes, done here: 2f5a09b